repose-aggregator/components/services/http-client-service/http-client-service-api/build.gradle

```diff
@@ -1,21 +1,10 @@
-
 dependencies {
     compile "org.apache.httpcomponents:httpclient"
+    compile "org.slf4j:jcl-over-slf4j"
+
+    compileOnly 'org.projectlombok:lombok'
 
     testCompile project(":repose-aggregator:commons:commons-test")
     testCompile "junit:junit"
     testCompile "org.scalatest:scalatest_$scalaMajDotMin"
 }
-
-jacocoTestCoverageVerification {
-    violationRules {
-        rules.forEach { rule ->
-            rule.limits.forEach { limit ->
-                if (limit.counter == 'INSTRUCTION' && limit.value == 'COVEREDRATIO') {
-                    // todo: set to the current coverage -- need more coverage, but we don't want the build to fail for the baseline
-                    limit.minimum = 0.02
```

Comment:
This can be increased, but it will need to remain and be at least `0.28`.

Reply:
Done. It's a shame it has to be set so low since all of the code that runs is covered...

Reply:
I know. We just need to move everything to Scala. Problem solved.